app/views/record/_basic_info.html.erb

```diff
@@ -6,7 +6,7 @@
 
 <% if @record.eds_cover_thumb_url.present? %>
 <div class="record-image">
-  <%= image_tag(@record.eds_cover_thumb_url, alt: "Cover for #{@record.title}") %>
+  <%= force_https_image_tag(@record.eds_cover_thumb_url, "Cover for #{@record.title}") %>
```

Comment:
Are we going to be happier if this acts like `(url, alt: alt)`, so that it has the same syntax as the `image_tag` it replaces?

Reply:
You know, I had the same thought and ignored it. I'll reconsider as both of us wondering that is definitely probably a signal we should change it.